genmod/commands/analyze.py

```diff
@@ -11,226 +11,220 @@
-                print_line = [variant_dict[variant_id].get(entry, '-') for entry in vcf_header]
-                f.write('\t'.join(print_line)+'\n')
+                    print("".join(word.ljust(column_width) for word in print_line))
+                print_line = [variant_dict[variant_id].get(entry, "-") for entry in vcf_header]
+                f.write("\t".join(print_line) + "\n")
             i += 1
-    
-    return
-
 
 
 ###           This is for analyzing the variants       ###
 
 
 def make_models(list_of_models):
     """Make a dictionary of the prefered models.
-        If no models are specified all are considered interesting."""
+    If no models are specified all are considered interesting."""
     model_set = set()
     # If no models are specified we allow all models
     if len(list_of_models) == 0:
-        list_of_models = ['AR', 'AD', 'X']
-    
+        list_of_models = ["AR", "AD", "X"]
+
     for model in list_of_models:
-        if 'AR' in model:
-            model_set.add('AR_hom')
-            model_set.add('AR_hom_dn')
-            model_set.add('AR_comp')
-            model_set.add('AR_comp_dn')
-        if 'AD' in model:
-            model_set.add('AD')
-            model_set.add('AD_dn')
-        if 'X' in model:
-            model_set.add('XR')
-            model_set.add('XR_dn')
-            model_set.add('XD')
-            model_set.add('XD_dn')
+        if "AR" in model:
+            model_set.add("AR_hom")
+            model_set.add("AR_hom_dn")
+            model_set.add("AR_comp")
+            model_set.add("AR_comp_dn")
+        if "AD" in model:
+            model_set.add("AD")
+            model_set.add("AD_dn")
+        if "X" in model:
+            model_set.add("XR")
+            model_set.add("XR_dn")
+            model_set.add("XD")
+            model_set.add("XD_dn")
     return model_set
 
 
-
 def remove_inacurate_compounds(compound_dict, family_id):
     """If the second variant in a compound pair does not meet the requirements they should not be considered."""
-    
+
     for variant_id in list(compound_dict.keys()):
         # Get the compounds for the variant
-        family_compounds = compound_dict[variant_id]['info_dict'].get('Compounds', None)
-        if compounds:
-            for family in family_compounds.split(','):
-                splitted_compounds = family.split(':')
-                if splitted_compounds[0] == family_id:
-                    compounds = splitted_compounds[1].split('|')
-                    compound_set = set(compounds) 
+        family_compounds = compound_dict[variant_id]["info_dict"].get("Compounds", None)
+        if family_compounds:
+            for family in family_compounds.split(","):
+                split_compounds = family.split(":")
+                if split_compounds[0] == family_id:
+                    compounds = split_compounds[1].split("|")
+                    compound_set = set(compounds)
```

Comment:
same here

Reply:
Same intention there I believe: assume only one matching family, and go with that. But whatever, I see what you are saying as well - it could work even better by just keeping adding if there happens to be more matches. Lets fix!